migrations/1_initial_migration.js

```diff
@@ -0,0 +1,5 @@
+const Migrations = artifacts.require("Migrations");
```

Comment:
Do we want to add truffle deployment script to `migrations/` directory?

Reply:
I'm on the fence about it. I actually strongly dislike their system and usually prefer to deploy by hand

Reply:
It would help in development and local testing. It would be also a reference for user for [linking mentioned in README](https://github.com/summa-tx/bitcoin-spv/blob/v2.0.0/README.md#why-is-there-a-library-and-a-delegate).

Production deployments will be done by hand as you prefer :smile:

Reply:
It won't help in local setup or testing because everything is stateless. This may change if we merge the Relay branch

The linking mentioned in the README must be done from their deploy scripts, and is not in any way useful in this library. And my deploy scripts are not the best place to document truffle linking features :P

Reply:
> It won't help in local setup or testing because everything is stateless.

Does it mean that `truffle migrate` command will pick up the contracts and deploy them?

Reply:
No, it means that the test `before` block can always deploy contracts with no additional scaffolding. No linking required, no constructor arguments necessary, basically there are no steps for the migration to perform